src/api.js

```diff
@@ -927,8 +928,8 @@ class VLWholeSlideMicroscopyImageViewer {
     }
     const allDrawOptions = Object.assign(defaultDrawOptions, customDrawOptions);
 
-    if (options.condition) {
-      allDrawOptions.condition = options.condition;
+    if (options.bindings) {
```

Comment:
Since we are adding more and more options to the constructor of the viewer class, we should add proper API documentation (using [JSDoc](https://github.com/jsdoc/jsdoc)). This doesn't have to occur via this PR, but it would be great if you could add a comment with a short description of `bindings`.

Reply:
Agree that long term we should flesh out the docs. Added a comment for now 👍